src/screens/LoginPage/LoginPage.tsx

```diff
@@ -57,6 +64,49 @@ function loginPage(): JSX.Element {
   const [showPassword, setShowPassword] = useState<boolean>(false);
   const [showConfirmPassword, setShowConfirmPassword] =
     useState<boolean>(false);
+  const [showAlert, setShowAlert] = useState<PasswordValidation>({
+    lowercaseChar: true,
+    uppercaseChar: true,
+    numericValue: true,
+    specialChar: true,
+  });
+
+  const passwordValidationRegExp = {
+    lowercaseCharRegExp: new RegExp('[a-z]'),
+    uppercaseCharRegExp: new RegExp('[A-Z]'),
+    numericalValueRegExp: new RegExp('\\d'),
+    specialCharRegExp: new RegExp('[!@#$%^&*()_+{}\\[\\]:;<>,.?~\\\\/-]'),
+  };
+  const handleLowercasePassCheck = (pass: string): void => {
+    if (passwordValidationRegExp.lowercaseCharRegExp.test(pass)) {
+      setShowAlert((prevAlert) => ({ ...prevAlert, lowercaseChar: false }));
+    } else {
+      setShowAlert((prevAlert) => ({ ...prevAlert, lowercaseChar: true }));
+    }
+  };
+
+  const handleUppercasePassCheck = (pass: string): void => {
```

Comment:
@MahendraDani are you sure these methods are not working like this ?

```ts

  const handleUppercasePassCheck = (pass: string): void => {
      setShowAlert((prevAlert) => ({ 
          ...prevAlert, 
          uppercaseChar: !passwordValidationRegExp.uppercaseCharRegExp.test(pass)
          }));
  };
```

Reply:
Let me give a try again

Reply:
@rishav-jha-mech You are right its working

Reply:
Push the changes, then your PR is ready to be merged

Reply:
@rishav-jha-mech 
Pushed the changes via https://github.com/PalisadoesFoundation/talawa-admin/pull/1376/commits/2a0d028c1812f8ccaae0c574055ac0ea52a02234